index.html

```diff
@@ -3,13 +3,46 @@
 <head>
     <meta charset="UTF-8">
     <title>Homepage</title>
-    </head>
+    <link rel="stylesheet" type="text/css" href="/public/css/index.css">
```

Comment:
I don't see this file! Also, I think it'd be good to use a css file for your html files instead of putting it directly on the html file!

Reply:
Agreed.  It looks like maybe the styles below were meant to go in the external `index.css`?